test/buildkitetestjson.jl

```diff
@@ -54,9 +55,23 @@ json_repr(io::IO, val::Any; indent::Int=0) = json_repr(io, string(val))
 # Test result processing
 
 function result_dict(testset::Test.DefaultTestSet, prefix::String="")
+    scope = if isempty(prefix)
+        testset.description == "Overall" ? "" : testset.description
+    else
+        join((prefix, testset.description), " → ")
+    end
     Dict{String, Any}(
         "id" => Base.UUID(rand(UInt128)),
-        "scope" => join((prefix, testset.description), '/'),
+        "scope" => scope,
+        "tags" => Dict{String, String}(
+            "job_label": get(ENV, "BUILDKITE_LABEL", "unknown"),
+            "job_id": get(ENV, "BUILDKITE_JOB_ID", "unknown"),
+            "os": string(Sys.KERNEL),
+            "arch": string(Sys.ARCH),
+            "julia_version": string(VERSION),
+            "testset": testset.description,
+            "prefix": prefix
```

Comment:
The syntax is wrong, I think this is what was meant:

```suggestion
            "job_label" => get(ENV, "BUILDKITE_LABEL", "unknown"),
            "job_id" => get(ENV, "BUILDKITE_JOB_ID", "unknown"),
            "os" => string(Sys.KERNEL),
            "arch" => string(Sys.ARCH),
            "julia_version" => string(VERSION),
            "testset" => testset.description,
            "prefix" => prefix
```

Reply:
Thanks! Brain in json mode